modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/image_selector/js/ImageSelector.js

```diff
@@ -51,16 +53,22 @@ const ImageSelector = ({
 }) => {
 	const [errorMessage, setErrorMessage] = useState('');
 	const [fileName, setFileName] = useState('');
-	const [image, setImage] = useState({
-		fileEntryId,
-		src: imageURL,
-	});
 	const [imageCropRegion, setImageCropRegion] = useState(
 		initialImageCropRegion
 	);
 	const [progressData, setProgressData] = useState();
 	const [progressValue, setProgressValue] = useState(0);
 
+	const [image, setImage] = useState({fileEntryId, src: imageURL});
```

Comment:
During the editing, this got moved down here (not really intentionally). Let me know if you want me to move it back where it was to make the overall diff smaller.

Reply:
No problem from my side

Reply:
Source formatter suffers seeying this not alphabetically sorted :P

Reply:
NP for me